js/src/data/reducer.js

```diff
@@ -46,10 +46,19 @@ function chain( state, basePath = '' ) {
 		}
 		return clone( value );
 	};
+	const combinBasePath = ( path ) => {
```

Comment:
❔

Is it needed to put this function inside of the `chain function` scope? I think we could just have this function outside of the `chain function` scope receiving two params: `basePath` and `path` thus it could be re-used easily in the future.

💅  I think there is a misspelling with the function's name `combinBasePath` should be `combineBasePath`

Reply:
Thanks for catching the typo. Fixed by https://github.com/woocommerce/google-listings-and-ads/pull/1184/commits/6534674b77c9cf934f7467e93b15d17c58485c41.

> Is it needed to put this function inside of the `chain function` scope? I think we could just have this function outside of the `chain function` scope receiving two params: `basePath` and `path` thus it could be re-used easily in the future.

I guess its reuse is likely to be relatively low. I'd prefer to move it out when the needs happen.

Reply:
Thanks for correcting this! 

> I guess its reuse is likely to be relatively low. I'd prefer to move it out when the needs happen.

Let's keep it as it is so we do not delay this PR and if in the future is needed we can just move it out.